Description: Sets default DSN
 The default DSN isn't working for the packaging, so we do that:
 +connection=sqlite:///var/lib/heat/neutrondb by default.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-04-08

Index: heat/etc/heat/heat.conf.sample
===================================================================
--- heat.orig/etc/heat/heat.conf.sample	2014-05-21 16:02:38.000000000 +0800
+++ heat/etc/heat/heat.conf.sample	2014-05-21 16:02:42.000000000 +0800
@@ -818,7 +818,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection=sqlite:////heat/openstack/common/db/$sqlite_db
+connection=sqlite:///var/lib/heat/neutrondb
 
 # The SQLAlchemy connection string used to connect to the
 # slave database (string value)
